.github/workflows/concrete_python_release_gpu.yml

```diff
@@ -84,67 +78,86 @@ jobs:
           options: >-
             -v ${{ github.workspace }}:/concrete
             -v ${{ github.workspace }}/build:/build
-            -v ${{ env.SSH_AUTH_SOCK }}:/ssh.socket
-            -e SSH_AUTH_SOCK=/ssh.socket
           shell: bash
           run: |
             set -e
 
-            rustup toolchain install nightly-2024-09-30
-            pip install mypy
             rm -rf /build/*
-            
+
             export PYTHON=${{ format('python{0}', matrix.python-version) }}
             echo "Using $PYTHON"
 
-            dnf -y install graphviz graphviz-devel
-            
             cd /concrete/frontends/concrete-python
             make PYTHON=$PYTHON venv
             source .venv/bin/activate
-            
+
             cd /concrete/compilers/concrete-compiler/compiler
             make BUILD_DIR=/build CCACHE=ON DATAFLOW_EXECUTION_ENABLED=OFF Python3_EXECUTABLE=$(which python) \
               CUDA_SUPPORT=ON TIMING_ENABLED=ON CUDA_PATH=${{ env.CUDA_PATH }} python-bindings
-            
+
             echo "Debug: ccache statistics (after the build):"
             ccache -s
 
             cd /concrete/frontends/concrete-python
 
             export COMPILER_BUILD_DIRECTORY="/build"
             make whl
-            
-            deactivate
 
+            deactivate
       - name: Upload wheel
         uses: actions/upload-artifact@50769540e7f4bd5e21e526ee35c689e35e0d6874 # v4.4.0
         with:
           name: ${{ format('wheel-{0}-linux-x86', matrix.python-version) }}
           path: frontends/concrete-python/dist/*manylinux*.whl
           retention-days: 3
+      - name: Slack Notification
+        if: ${{ failure() }}
+        continue-on-error: true
+        uses: rtCamp/action-slack-notify@4e5fb42d249be6a45a298f3c9543b111b02f7907
+        env:
+          SLACK_COLOR: ${{ job.status }}
+          SLACK_MESSAGE: "build-linux-x86 (${{matrix.python-version}}) finished with status: ${{ job.status }}. (${{ env.ACTION_RUN_URL }})"
```

Comment:
Maybe enhance the message with a simple addition `"build-linux-x86 (Python: ${{matrix.python-version}}) [...]` to know that we're testing against Python.

Reply:
that implicit for concrete people, this is like that on other place